app/models/role.rb

```diff
@@ -3,7 +3,7 @@
 # "moderator" for an instance of a model (i.e., an object), a model class,
 # or without any specification at all.
 class Role < ApplicationRecord
-  has_many :roles_users
+  has_many :roles_users, dependent: :destroy
```

Comment:
What requires this change btw?

Reply:
Nothing specific -- I thought it would be better to have it than not, since a) a missing `dependent: :destroy` on the user model side of roles_users was causing callback issues, and b) we were talking about possibly removing the translation_admin role, but I can definitely leave it for that instead.